src/QuadForm/Misc.jl

```diff
@@ -862,9 +862,9 @@ function _find_quaternion_algebra(b, P, I)
 
   found = false
   U, h = unit_group(R)
-  sign_vector = g -> begin
+  function sign_vector(g)
```

Comment:
why not anonymous?

Reply:
My guess is that anonymous functions don't get a recognizable name in Jet analysis, but no real idea